roles/dhcp-server/templates/local.conf.j2

```diff
@@ -8,3 +8,6 @@ dhcp-boot=pxelinux.0
 dhcp-authoritative
 enable-tftp
 tftp-root=/srv/tftp
+no-hosts
```

Comment:
I still don't like this change, I find it useful to be able to ssh into machines by hostname, on the video network.